container/os-autoinst_dev/Dockerfile

```diff
@@ -97,6 +97,7 @@ RUN zypper in -y -C \
        'perl(IPC::Open3)' \
        'perl(IPC::Run::Debug)' \
        'perl(IPC::System::Simple)' \
+       'perl(Inline::Lua)' \
```

Comment:
Will this work if perl-Inline-Lua is only in devel:languages:perl at this point?

Reply:
I created a comment

Reply:
Updated the comment

Reply:
I fixed the factory link with:
```
% osc changedevelrequest openSUSE:Factory perl-Inline-Lua devel:languages:perl perl-Inline-Lua -m "assign a devel project"
1282063
% osc linkpac openSUSE:Factory perl-Inline-Lua devel:languages:perl perl-Inline-Lua -f
Creating _link... Done.
```
https://build.opensuse.org/requests/1282063